README.md

```diff
@@ -13,6 +13,23 @@ As this package is just a wrapper around `flint`, you will need something to
 actually run it. By default the [Linter][] package will be installed for you to
 fill this need.
 
+# Considerations
+
+By default the linter status bar shows counts for the entire project and the panel shows the messages for the current file.
```

Comment:
How about something like this to replace this entire section?
```markdown
Since `flint` reports messages about the project as a whole there is no file to associate them to, so we mark them as coming from a "fake" `flint-messages` file. Unfortunately under the default settings for `linter-ui-default`, this means the messages from this provider will only ever show up in the status bar counts.

In order to show these messages you will need to change `linter-ui-default`'s Panel to represent the entire project in the settings.
```

Reply:
Perfect!